.github/workflows/build-test-package.yml

```diff
@@ -305,11 +314,19 @@ jobs:
         sudo rm -rf "$AGENT_TOOLSDIRECTORY"
         df -h
 
+    - name: 'Fetch build dependencies'
+      shell: bash
+      run: |
+        sudo apt install zstd
```

Comment:
Earlier we use `apt-get` instead of `apt`.  Probably, it is okay to mix and match, but we could nonetheless change it for consistency.

Reply:
Updated